examples/extensions/deployers/sources/custom_deployer_sources.rst

```diff
@@ -0,0 +1,80 @@
+.. examples_extensions_deployers_sources:
+
+Copy sources from all your dependencies
+=======================================
+
+
+
+Please, first of all, clone the sources to recreate this project. You can find them in the
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ in GitHub:
```

Comment:
```suggestion
`examples2 repository <https://github.com/conan-io/examples2>`_ in GitHub:
```
the name of repository doesn't match

Reply:
You're right, but this banner is used elsewhere as-is, so I'm leaving it like this just to keep them all consistent :)